### ISSUE TITLE
Implement get_bytes_webp function

### ISSUE THREAD

**Issue by hyperparameters** [fineshopdesign/cf-wasm#5]
Hello Thankyou for the making photon compatible with Cloudflare Workers, 
`silvia-odwyer/photon` has implemented `get_bytes_webp` function that converts image to webp.

It would be very helpful if you can implement this function

**Comment by fineshop** [comment]
Thank you for opening an issue.
Can you please share any reference to this function?
I can't see this function in `silvia-odwyer/photon`.

**Comment by hyperparameters** [comment]
the function is available in the following file

`file path silvia-odwyer/photon/crate/src/lib.rs`
```
pub fn get_bytes_webp(&self) -> Vec<u8> {
        let mut img = helpers::dyn_image_from_raw(self);
        img = ImageRgba8(img.to_rgba8());
        let mut buffer = vec![];
        let out_format = image::ImageOutputFormat::WebP;
        img.write_to(&mut Cursor::new(&mut buffer), out_format)
            .unwrap();
        buffer
    }
```

**Comment by fineshop** [comment]
Thank you! 😀
But they are recent commits and not yet released.
If you would like me to fork the latest repo and build it for workers, I can do it 🙂.

**Comment by hyperparameters** [comment]
I think it is safe to do that, 👍

**Comment by fineshop** [comment]
Thank you!
I shall make changes when I get time. 😃

**Comment by fineshop** [comment]
Implemented get_bytes_webp in #6

**Comment by fineshop** [comment]
Implemented get_bytes_webp in #6
